commands/test/test-command.ts

```diff
@@ -153,22 +153,62 @@ export const sendQuestion = async (interaction: any) => {
         console.log(userResponses);
         console.log(interaction.user.id);
 
-
-
         const bestMatch = await findMatchingUser(interaction.user.id, userResponses);
-
+       
         if (bestMatch) {
-            interaction.user.send(`Dein bester Gesprächspartner ist: **@${bestMatch.username}**.`);
-            interaction.user.send("Als nächstes schreibst du deinem Partner, indem du auf seinen Namen auf dem Contraversum-Server klickst 👆 und ihm eine Nachricht sendest.");
-            interaction.user.send("Dies sind drei Fragen bei denen ihr euch unterscheidet:");
-
-            // Send the best match that they have been matched with the user
-            const bestMatchUser = await client.users.fetch(bestMatch.userId);
-            if (bestMatchUser) {
-                bestMatchUser.send(`Hey 👋, du wurdest mit: **@${interaction.user.username}** gematched.`);
-                bestMatchUser.send("Ihr habt euch bei diesen drei Fragen beispielsweise unterschieden:");
+            const guildId = process.env.GUILD_ID;
+            if (!guildId) {
+                console.error('GUILD_ID is not found');
+                return;
+            }
+            const guild: Guild | undefined = client.guilds.cache.get(guildId);
+            if (!guild) {
+                console.error('Guild not found');
+                return;
+            }
+
+            const member = guild.members.cache.get(interaction.user.id);
```

Comment:
it might be a bit confusing since we have a 'member' and bestMatch.memberId maybe a less confusing variable name would be better?